app/presenters/licences/view-licence.presenter.js

```diff
@@ -71,7 +71,8 @@ function go (licence, licenceAbstractionConditions) {
     registeredTo,
     sourceOfSupply: abstractionDetails.sourceOfSupply,
     startDate: formatLongDate(startDate),
-    warning: _generateWarningMessage(ends)
+    warning: _generateWarningMessage(ends),
+    roles: _getAuthRoles(auth)
```

Comment:
Started to get a sonar cloud error - https://sonarcloud.io/project/issues?resolved=false&severities=MAJOR&sinceLeakPeriod=true&branch=view-license-role-based-tabs-for-bills-and-charge-information&id=DEFRA_water-abstraction-system&open=AY8pIrHhp7fcpGn2GYJe

this did not happen when the notification work was done.